README.md

```diff
@@ -248,6 +264,130 @@ _Note: The -K is used to prompt you for your password for sudo (if you require o
        The -k is used to prompt you for your ssh password can hit enter if using -K and they are the same<br>
        Instead of -k you could use --private-key=<absolute_path_to_ssh_private_key>_
 
+### Example 7a: OpenShift cluster instance endpoint + Helper infra OS templates + OS templates from some project
+
+ 1. Install on an OpenShift cluster endpoint.
+ 2. Setup helper infrastructure containers such as:
+    1. Jenkins
+    2. Jenkins-slave
+    3. ansible-execeutor
+    4. linchpin-executor
+    5. container-tools<br>
+    _Note: Exclude other helper containers influxdb and grafana_
+ 3. Setup OpenShift s2i templates from the -e project_repo=https://github.com/arilivigni/my-project-example
+    1. Override the project_repo with another one then the default in global.yml
+    2. OpenShift project -e openshift_project=ari-my-project 
+ 4. Modify my container tags with the default tag. tag=stable
+ 5. Don't setup or start minishift
+ 6. Don't run pre-reqs
+ 7. Dont setup nested virt
+
+```
+    ansible-playbook -vv -i "localhost," contra-env-setup/playbooks/setup.yml \
+    -e user=cloud-user -e project_repo=https://github.com/arilivigni/my-project-example \
+    -e openshift_project=ari-ci-pipeline -e setup_minishift=false -e start_minishift=true \
```

Comment:
whats the point of having `start_minishift=true` if `setup_minishift=false` ?

Reply:
One starts minishift if it has already been setup but may be stopped and the other sets it up.  If you want more detail please look into the code.  This functionality has been there for a while and introduced by the CVP team.